CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 	- Removed "last user" cache to avoid overwriting anonymous user cache during init [SDKCF-6409]
 - Improvements:
 	- Created new test scheme to combine unit test and UI test results [SDKCF-6356]
+    - Added a new file for EventType unit test to increase code coverage [SDKCF-6378]   
```

Comment:
@ts-robih-robihamanto Please fix this indentation.

Reply:
Fixed [here](https://github.com/rakutentech/ios-inappmessaging/pull/254/commits/591b7474ac3645273c7e7763c86fa6ab90e5cd27).